<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1105</width>
    <height>575</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="1" column="0">
     <widget class="QTabWidget" name="uitabwidget_2">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>MediaPlayer</string>
       </attribute>
       <widget class="QWidget" name="verticalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>40</y>
          <width>351</width>
          <height>79</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_artist">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color:rgb(255,195,50)</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_title">
           <property name="styleSheet">
            <string notr="true">color:rgb(255, 255, 255)</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="5,5">
           <property name="spacing">
            <number>15</number>
           </property>
           <property name="sizeConstraint">
            <enum>QLayout::SetDefaultConstraint</enum>
           </property>
           <item>
            <widget class="QSlider" name="horizontalSlider_2">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="styleSheet">
              <string notr="true">QSlider::groove:horizontal {
border: 0px solid #bbb;
background: white;
height: 10px;
border-radius: 4px;
}

QSlider::sub-page:horizontal {
background: qlineargradient(x1: 0, y1: 0,    x2: 0, y2: 1,
    stop: 0 #ffc332, stop: 1 #e5af2d);
background: qlineargradient(x1: 0, y1: 0.2, x2: 1, y2: 1,
    stop: 0 #e5af2d, stop: 1 #ffc946);
border: 0px solid #333;
height: 10px;
border-radius: 4px;
}

QSlider::add-page:horizontal {
background: #99000000;
border: 0px solid #777;
height: 10px;
border-radius: 4px;
}

QSlider::handle:horizontal {
background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
    stop:0 #eee, stop:1 #ccc);
border: 0px solid #777;
width: 13px;
margin-top: -2px;
margin-bottom: -2px;
border-radius: 4px;
}

QSlider::handle:horizontal:hover {
background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
    stop:0 #fff, stop:1 #ddd);
border: 1px solid #444;
border-radius: 4px;
}

QSlider::sub-page:horizontal:disabled {
background: #bbb;
border-color: #999;
}

QSlider::add-page:horizontal:disabled {
background: #eee;
border-color: #999;
}

QSlider::handle:horizontal:disabled {
background: #eee;
border: 1px solid #aaa;
border-radius: 4px;
}
</string>
             </property>
             <property name="value">
              <number>67</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_duration_2">
             <property name="sizePolicy">
              <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>75</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="sizeIncrement">
              <size>
               <width>400</width>
               <height>0</height>
              </size>
             </property>
             <property name="baseSize">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
             <property name="layoutDirection">
              <enum>Qt::LeftToRight</enum>
             </property>
             <property name="styleSheet">
              <string notr="true">color:rgb(255, 255, 255)</string>
             </property>
             <property name="text">
              <string>00:00/ 00:00</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
             <property name="wordWrap">
              <bool>false</bool>
             </property>
             <property name="margin">
              <number>5</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>40</y>
          <width>81</width>
          <height>81</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>Resource/img/freemp.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="margin">
         <number>5</number>
        </property>
       </widget>
       <widget class="QLabel" name="label_bgr">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>40</y>
          <width>471</width>
          <height>131</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color : qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(56, 64, 68, 255), stop:1 rgba(29, 33, 37, 255))</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QWidget" name="horizontalLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>120</y>
          <width>471</width>
          <height>52</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QPushButton" name="pushButton_shuffle">
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>Resource/img/btn_shuffle_off.png</normaloff>Resource/img/btn_shuffle_off.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_prev">
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>Resource/img/btn_previous.png</normaloff>Resource/img/btn_previous.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_play">
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>Resource/img/btn_play.png</normaloff>Resource/img/btn_play.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_next">
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>Resource/img/btn_next.png</normaloff>Resource/img/btn_next.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_repeat">
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>Resource/img/btn_repeat_off.png</normaloff>Resource/img/btn_repeat_off.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QListView" name="listView">
        <property name="geometry">
         <rect>
          <x>630</x>
          <y>20</y>
          <width>401</width>
          <height>451</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QListView {
background-color:
qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(56, 64, 68, 255), stop:1 rgba(29, 33, 37, 255))
}
QScrollBar {
     border: none;
     background: #494949;
     height: 6px;
     width: 6px;
     margin: 0px;
}

QScrollBar::handle {
     background: #DBDBDB;
     min-width: 20px;
     min-height: 20px;
}

 QScrollBar::add-line,  QScrollBar::sub-line {
     background: none;
     border: none;
}

 QScrollBar::add-page, QScrollBar::sub-page {
     background: none;
}
</string>
        </property>
        <property name="dragDropOverwriteMode">
         <bool>false</bool>
        </property>
        <property name="dragDropMode">
         <enum>QAbstractItemView::NoDragDrop</enum>
        </property>
        <property name="defaultDropAction">
         <enum>Qt::IgnoreAction</enum>
        </property>
       </widget>
       <widget class="QWidget" name="verticalLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>190</y>
          <width>554</width>
          <height>150</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QListWidget" name="playlists"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QPushButton" name="SkipPrevious">
             <property name="text">
              <string>&lt;</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="Play">
             <property name="text">
              <string>Play/Pause</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="SavePlaylist">
             <property name="text">
              <string>Save Playlist</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="LoadPlaylist">
             <property name="text">
              <string>Load Playlist</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="SkipNext">
             <property name="text">
              <string>&gt;</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QSlider" name="horizontalSlider">
        <property name="geometry">
         <rect>
          <x>420</x>
          <y>10</y>
          <width>160</width>
          <height>16</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QVideoWidget" name="vidwid" native="true">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>350</y>
          <width>481</width>
          <height>121</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>380</y>
          <width>91</width>
          <height>27</height>
         </rect>
        </property>
        <property name="text">
         <string>RadioStation</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>410</y>
          <width>91</width>
          <height>27</height>
         </rect>
        </property>
        <property name="text">
         <string>FindMusic ./</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_4">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>350</y>
          <width>91</width>
          <height>27</height>
         </rect>
        </property>
        <property name="text">
         <string>OpenFolder</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_5">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>440</y>
          <width>91</width>
          <height>27</height>
         </rect>
        </property>
        <property name="text">
         <string>OpenFile</string>
        </property>
       </widget>
       <zorder>label_bgr</zorder>
       <zorder>verticalLayoutWidget</zorder>
       <zorder>label_3</zorder>
       <zorder>horizontalLayoutWidget_2</zorder>
       <zorder>listView</zorder>
       <zorder>verticalLayoutWidget_2</zorder>
       <zorder>horizontalSlider</zorder>
       <zorder>vidwid</zorder>
       <zorder>pushButton</zorder>
       <zorder>pushButton_2</zorder>
       <zorder>pushButton_4</zorder>
       <zorder>pushButton_5</zorder>
      </widget>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>ShoutCast</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>ShoutCast</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="3" column="2">
            <widget class="QLineEdit" name="stationSearch"/>
           </item>
           <item row="4" column="3">
            <widget class="QPushButton" name="refreshbtn">
             <property name="text">
              <string>refresh</string>
             </property>
            </widget>
           </item>
           <item row="3" column="4" rowspan="3">
            <widget class="QTreeWidget" name="treeWidget">
             <column>
              <property name="text">
               <string>1</string>
              </property>
             </column>
             <item>
              <property name="text">
               <string>New Item</string>
              </property>
              <item>
               <property name="text">
                <string>New Subitem</string>
               </property>
               <item>
                <property name="text">
                 <string>New Subitem</string>
                </property>
                <item>
                 <property name="text">
                  <string>New Subitem</string>
                 </property>
                </item>
               </item>
              </item>
             </item>
            </widget>
           </item>
           <item row="1" column="2" colspan="2">
            <widget class="QListWidget" name="stationurls"/>
           </item>
           <item row="1" column="1" rowspan="5">
            <widget class="QListWidget" name="txtNames"/>
           </item>
           <item row="5" column="2">
            <widget class="QPushButton" name="chooseStationbtn">
             <property name="text">
              <string>ChooseStation</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0" rowspan="5">
            <widget class="QListWidget" name="txtIds">
             <property name="sizeAdjustPolicy">
              <enum>QAbstractScrollArea::AdjustToContents</enum>
             </property>
            </widget>
           </item>
           <item row="4" column="2">
            <widget class="QPushButton" name="pushButton_3">
             <property name="text">
              <string>Search</string>
             </property>
            </widget>
           </item>
           <item row="2" column="2" colspan="2">
            <widget class="QComboBox" name="comboBox">
             <item>
              <property name="text">
               <string>Alternative</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Blues</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Classical</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Country</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Easy Listening</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Electronic</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Folk</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Themes</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Rap</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Inspirational</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>International</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Jazz</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Latin</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Metal</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>New Age</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Decades</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Pop</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>R&amp;B and Urban</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Reggae</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Rock</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Seasonal and Holiday</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Soundtracks</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Talk</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Misc</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Public Radio</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1105</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMain">
    <property name="title">
     <string>&amp;Main</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionNew"/>
    <addaction name="actionClose"/>
    <addaction name="actionRestore"/>
    <addaction name="actionExit"/>
    <addaction name="actionSave"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>Abo&amp;ut</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuPlaylist">
    <property name="title">
     <string>Playlist</string>
    </property>
    <addaction name="actionScan"/>
    <addaction name="actionClear"/>
   </widget>
   <addaction name="menuMain"/>
   <addaction name="menuAbout"/>
   <addaction name="menuPlaylist"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>&amp;New</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>&amp;Close</string>
   </property>
  </action>
  <action name="actionRestore">
   <property name="text">
    <string>&amp;Restore</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionScan">
   <property name="text">
    <string>&amp;Scan</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="text">
    <string>&amp;Clear</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVideoWidget</class>
   <extends>QWidget</extends>
   <header>QVideoWidget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
